src/components/admin/ActionButtons.vue

```diff
@@ -1,19 +1,79 @@
 <template>
-  <q-td :props="props" class="text-right">
-    <q-btn color="green" class="q-mx-sm">Approve</q-btn>
+  <div class="flex justify-center">
     <q-btn
+      round
+      flat
+      color="green"
+      class="q-mx-sm"
+      icon="check"
+      @click="approveApplication"
+    >
+      <q-tooltip>Approve</q-tooltip>
+    </q-btn>
+    <q-btn
+      round
+      flat
       color="red"
+      icon="close"
       class="q-mx-sm"
-      @click="TheDialogs.emit({ type: 'applicationRejectDialog', arg: {} })"
-      >Reject</q-btn
+      @click="rejectApplication"
     >
-  </q-td>
+      <q-tooltip>Reject</q-tooltip>
+    </q-btn>
+    <q-btn round flat icon="more_horiz" @click="viewApplication">
+      <q-tooltip>View Application</q-tooltip>
+    </q-btn>
+  </div>
 </template>
 
 <script setup>
 import { TheDialogs } from 'src/dialogs/the-dialogs';
+import { TheWorkflows } from 'src/workflows/the-workflows';
+import { Notify } from 'quasar';
 
-defineProps({
+const props = defineProps({
   props: Object,
 });
+
+function viewApplication() {
+  TheDialogs.emit({
+    type: 'viewStudentApplicationDialog',
+    arg: {
+      payload: props.props,
+    },
+  });
+}
+
+function approveApplication() {
+  Notify.create({
+    message: 'Are you sure?',
+    position: 'center',
+    actions: [
+      {
+        name: 'yes',
+        label: 'yes',
+        handler() {
+          TheWorkflows.emit({
+            type: 'approveApplication',
+            arg: {
+              payload: { ...props.props, data: { ...props.props.data } },
+            },
+          });
+        },
+      },
+      {
+        label: 'No',
+      },
+    ],
+  });
```

Comment:
You may remove this, as this is temporary implementation. View Application is sufficient.